### ISSUE TITLE
Thanks for this - any chance of a win32 build?

### ISSUE THREAD

**Issue by BDD2** [codekiddy2/Visual-Studio-gtkmm#3]
Your work on this is much appreciated!
I would like to update software I am writing to the newer (and perhaps more bugfree) version of gtkmm and gtk3 that you have compiled here, but I find myself needing a win32 (x86) build. This is due to certain 32 bit libraries I am linking (lua, alut, openal). I could probably chase 64 bit libraries down, but we are also still using win32 at work so I guess I am stuck with a 32 bit build for now. Is this something that would be easy for you to add to your repository?

Again, thanks -- it's good to see that gtkmm on windows hasn't been forgotten.

**Comment by codekiddy2** [comment]
Hi BDD2, I'm sorry for late reply due to my inactivity for some 4 months now...

The reposotory does contain both x32 and x86 build configurations, both release and debug versions...
I worked mostly on x64 debug builds, but other versions have been tested and compiled without no problem.

You can find both configurations (x86, x64) in VS solutions for each dependency package as well as other projects.

**Comment by cingelvladimir** [comment]
Just came by to say thanks... This ready to use package saved me a LOT of time...

**Comment by cingelvladimir** [comment]
But i would even donate (via paypal or whatever) to have x64 release... not only debug as download ready (compiled) folder... :-)

**Comment by codekiddy2** [comment]
Oh I see now you guys talk about pre-built binaries from release section... yes the release section contains debug x64 package only.

I consider this project not fully finished, and it does require a lot of time to test and finalize final build result, and since I worked in debug mode for that matter that's what I included into release...

 I will include build results of other versions too one day but can't say when.
You will know when the work is in progress by looking at pulse.
https://github.com/codekiddy2/Visual-Studio-gtkmm/pulse

until that time your best bet is to build all from my repo or use other peoples work.
for example these guys seem to be active:
https://github.com/wingtk/gtk-win32

Thank you a lot cingelvladimir for considering a donation! I'm really busy right now with other things and don't even have stuff ready on my PC to simply compile.

Thank you for understanding!
